.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  release:
+    types:
+      - published
+name: Documentation
+jobs:
+  build:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6'
+
+    - name: Build Library and Docs
+      run: ./scripts/build
```

Comment:
This calls bundle install and the yard doc generator. Do I need the bundle install part?

Reply:
If doc generation requires the bundle, then yep